index.bs

```diff
@@ -297,32 +297,37 @@ mechanism for metadata discovery.
 Transport and metadata discovery with QUIC {#transport}
 =======================================================
 
-If a listening agent wants to connect to an advertising agent, or to
-learn further metadata about it, it initiates a [[QUIC]] connection to
-the IP and port from the SRV record.  Prior to authentication, a message may be
-exchanged (such as further metadata), but such info should be treated as
-unverified (such as indicating to a user that a display name of an
-unauthenticated agent is unverified).
+If an agent wants to connect to or learn further metadata about another agent,
+it initiates a [[QUIC]] connection to the IP and port from the SRV record.
+Prior to authentication, a message may be exchanged (such as further metadata),
+but such info should be treated as unverified (such as indicating to a user that
+a display name of an unauthenticated agent is unverified).
 
 To learn further metadata, an agent may send an agent-info-request
-message (see [[#appendix-a]]) and receive back an agent-info-response message.  The
-messages may contain the following information with the following meaning:
+message (see [[#appendix-a]]) and receive back an agent-info-response message.
+Any agent may send this request to learn about the capabilities of
+another device.
+
+The agent-info-response messages contain the following properties:
 
 : display-name (required)
-:: The display name of the responding agent intended
-    to be displayed to a user by the requesting agent.  If the responding agent
-    is not yet authenticated, the requesting agent should make UI affordance for
-    indicating to the user that the display name is not yet verified.  If the
-    responding agent changes its display name, the requesting agent should
-    make UI affordance for indicating to the user that the display name has
-    changed.
+:: The display name of the agent, intended to be displayed to a user by the
+     requester. The requester should indicate through the UI if the device
+     is not authenticated or if the display name changes.
 
 : model-name (optional)
 :: If the agent is a hardware device, the model name of
     the device.  This is used mainly for debugging purposes, but may be
     displayed to the user of the requesting agent.
 
-<!-- TODO: Add device type and/or capabilities -->
+: supports-audio (optional)
+:: The agent has to indicate that it supports audio streaming. If false
```

Comment:
It think we should show support for audio and video in general, not specific to streaming (such as for the presentation API).  This is mostly to know if we should show an a/v icon or just an audio icon.  

Also, the name should probably be "receives-audio" and "receives-video".

Reply:
Agreed.